### ISSUE TITLE
Documentation regarding the items order of an owned collection

### ISSUE THREAD

**Issue by Rorotu** [dotnet/efcore#19413]
I couldn't find information in the documentation regarding the items order of an owned collection. Specifically, I am looking for this two pieces of information : 

1.Are the items of the collection inserted in the same order that they appear in the collection (e.g. List<Object>) ? Will the key (e.g. implicit keys generated) of the subsequent element always be greater than the previous one ?
2.When querying the owner, will the generated sql query will always have an order by clause on the key of the owner and the keys, implicit or not, of the items of the owned collection ?

Basically, if this is confirmed, I wouldn't have to take care of the items order because it will be the same as in the collection when I query the owner.

### Further technical details

EF Core version: 3.1
Database provider: Microsoft.EntityFrameworkCore.SqlServer

**Comment by ajcvickers** [comment]
@Rorotu Order of entities in navigation properties should not be considered stable. Issue #9067 is tracking support for this.
